### ISSUE TITLE
[Q] Training embeddings only for newly added tokens

### ISSUE THREAD

**Issue by creatorrr** [UKPLab/sentence-transformers#2840]
Hey @tomaarsen -

I have a somewhat unusual experiment that I wanted to try where I would like to add some special tokens to a pretrained sentence-transformer model (say BAAI/bge-base-en-v1.5), and then finetune the model with a dataset for **only** training the added tokens and leaving the rest untouched.

More concretely, here's what I wanted to try:
- Taking conceptnet relations like `AtLocation`, `IsA`, `AntonymOf`
- Add corresponding tokens for them in the bge tokenizer like `<|AtLocation|>` etc
- Use `(head, relation, tail)` triplets from the conceptnet dataset
- Minimize loss for `norm(embed(head) + embed(relation_token) - embed(tail))`

Is there a way to easily try this out using _sentence-transformers_? Couldn't find it on the docs. Thanks again for the awesome lib! 🤗

**Comment by creatorrr** [comment]
Found my answer! Closing for now :)
